src/main/java/games/strategy/engine/pbem/ForumPosterComponent.java

```diff
@@ -25,7 +25,7 @@
   protected GameData data;
   // protected JLabel m_actionLabel;
   protected IPlayerBridge bridge;
-  protected PBEMMessagePoster m_poster;
+  protected PBEMMessagePoster poster;
```

Comment:
@RoiEXLab When you renamed all the fields in this class some time ago, you skipped this field.  Was there any particular reason?  I didn't notice any special usage that would preclude renaming it.

Reply:
I can't remember any and I don't think so either...
Probably just a mistake on my side.